greenboot.spec

```diff
@@ -0,0 +1,103 @@
+%global github_owner    LorbusChris
+%global github_project  greenboot
+%global github_branch   master
+%global build_timestamp %(date +"%Y%m%d%H%M%%S")
+
+Name:               greenboot
+Version:            0.1
+Release:            1%{?dist}
+Summary:            Generic Health Check Framework for systemd
+License:            LGPLv2+
+URL:                https://github.com/%{github_owner}/%{github_project}
+Source0:            https://github.com/%{github_owner}/%{github_project}/archive/%{github_branch}.tar.gz
+
+BuildArch:          noarch
+BuildRequires:      systemd
+%{?systemd_requires}
+Requires:           systemd
+
+%description
+%{summary}.
+
+%package notifications
+Summary:            Notification scripts for greenboot
+
+%description notifications
+Notification scripts for greenboot
+
+%package ostree
+Summary:            OSTree specific scripts for greenboot
+
+%description ostree
+OSTree specific scripts for greenboot
+
+%prep
+%setup -n %{github_project}-%{version}
+
+%build
+
+%install
+install -Dpm 0755 usr/libexec/greenboot/greenboot.sh %{buildroot}%{_libexecdir}/%{name}/%{name}.sh
+install -Dpm 0644 usr/lib/systemd/system/greenboot.target %{buildroot}%{_unitdir}/greenboot.target
+install -Dpm 0644 usr/lib/systemd/system/greenboot-healthcheck.service %{buildroot}%{_unitdir}/greenboot-healthcheck.service
+install -Dpm 0644 usr/lib/systemd/system/greenboot.service %{buildroot}%{_unitdir}/greenboot.service
+install -Dpm 0644 usr/lib/systemd/system/redboot.service %{buildroot}%{_unitdir}/redboot.service
+mkdir -p %{buildroot}%{_sysconfdir}/%{name}.d/check/required
+install -Dpm 0755 etc/greenboot.d/check/required/00_required_scripts_start.sh %{buildroot}%{_sysconfdir}/%{name}.d/check/required/00_required_scripts_start.sh
+mkdir    %{buildroot}%{_sysconfdir}/%{name}.d/check/wanted
+install -Dpm 0755 etc/greenboot.d/check/wanted/00_wanted_scripts_start.sh %{buildroot}%{_sysconfdir}/%{name}.d/check/wanted/00_wanted_scripts_start.sh
+mkdir    %{buildroot}%{_sysconfdir}/%{name}.d/green
+install -Dpm 0755 etc/greenboot.d/green/00_greenboot_notification.sh %{buildroot}%{_sysconfdir}/%{name}.d/green/00_greenboot_notification.sh
+mkdir    %{buildroot}%{_sysconfdir}/%{name}.d/red
+install -Dpm 0755 etc/greenboot.d/red/00_redboot_notification.sh %{buildroot}%{_sysconfdir}/%{name}.d/red/00_redboot_notification.sh
+install -Dpm 0755 etc/greenboot.d/red/98_ostree_rollback.sh %{buildroot}%{_sysconfdir}/%{name}.d/red/98_ostree_rollback.sh
+install -Dpm 0755 etc/greenboot.d/red/99_reboot.sh %{buildroot}%{_sysconfdir}/%{name}.d/red/99_reboot.sh
```

Comment:
split this out into its own package, so we don't force everyone to reboot on a red boot.

Reply:
that's done